### PR TITLE
Feature/bloglayout

### PR DESCRIPTION
**Pull request by Wolfr**
Start working on a blog layout.

Screenshot 2020-03-28 at 11 57 27
Screenshot 2020-03-28 at 11 57 33

### CODE REVIEW

**Comment by Wolfr** [comment]
I did a bit more work on this branch... @jakobrosenberg do you want to collab on this branch?

* The blog posts need to be shown in the reverse
* Correct the dates - how do we make it easy to deal with dates?
* First blog = just static content like this?

**Comment by jakobrosenberg** [comment]
> I did a bit more work on this branch... @jakobrosenberg do you want to collab on this branch?

Sure, I'll use this branch for implementing the backend. Will probably be Firebase.

**Comment by Wolfr** [comment]
I added some markdown files that can serve as an example.

Now I am just wondering how we can parse those to real blog posts.

File: src/pages/blog/index.svelte [hunk 1]
```diff
@@ -26,10 +20,13 @@
       <li class="c-blogpost-list__item">
         <article class="c-blogpost">
           <header>
-            <h2><a href="/blog/detail">{post.title}</a></h2>
-            <p>Written by @{post.author}</p>
+            <!-- @todo link to /blog/2020-04-06/a-start - so without the .md file name -->
```
Comment (Wolfr) [inline src/pages/blog/index.svelte:23]: Here I have no idea how to start doing these kinds of things

**Comment by Wolfr** [comment]
I've added some sample blog posts and managed to loop over them (thanks to the help in the chat), but no idea how to go further... does this need a change to Routify's internals to work?

**Pull request merged**